### DISCUSSION TITLE
ClickSmart+  CMA30036 & 35

### DISCUSSION THREAD

**Discussion by kernowkoi** [Koenkk/zigbee2mqtt#22764] [Devices]
Hi

Does any one know if the ClickSmart+  CMA30036 & 35  act as Zigbee routers for mesh extension?

Thanks

**Comment by iantait** [comment]
I have several CMA30036 - they are routers, and work well for me.

**Comment by kernowkoi** [comment]
Grand thanks will get some to try out
